hafnian/_hafnian.py

```diff
@@ -49,8 +49,9 @@ def input_validation(A, tol=1e-12):
     if np.isnan(A).any():
         raise ValueError("Input matrix must not contain NaNs.")
 
-    if np.linalg.norm(A - A.T) >= tol:
-        raise ValueError("Input matrix must be symmetric.")
+    if check_symmetry:
+        if np.linalg.norm(A - A.T) >= tol:
+            raise ValueError("Input matrix must be symmetric.")
```

Comment:
That is just to make sure that the input matrix is symmetroc. It raises error if not.

Reply:
Oh, I meant the `check_symmetry` flag. Is there a reason to turn off this check?

Reply:
There was because I thought multidimensional hermite polynomials are defined for any matrix. In reality they only make sense for symmetric matrices. Will remove this option